### PR TITLE
Rename app and update notifications in tray

### PR DESCRIPTION
**Pull request by AKalinich-Luxoft**
Now notifications in a tray get an updated text when SDL core state changes
Also changed app name

### CODE REVIEW

File: app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java [hunk 1]
```diff
@@ -76,25 +80,33 @@ public void run() {
...
+            }
         } catch (InterruptedException e) {
             e.printStackTrace();
         }
 
         final Intent intent = new Intent(ON_SDL_SERVICE_STOPPED);
         getApplicationContext().sendBroadcast(intent);
 
+        stopForeground(true);
+
```
Comment (VadymLuchko) [inline app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: @AKalinich-Luxoft 
for a complete stop probably add stopSelf () :
https://stackoverflow.com/questions/20857120/what-is-the-proper-way-to-stop-a-service-running-as-foreground

  stopForeground(true);
  stopSelf()

File: app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java [hunk 2]
```diff
@@ -106,17 +118,44 @@ private void showToastMessage(String message) {
...
     }
 
+    private void updateServiceNotification(String text) {
+        Notification notification = getServiceNotification(text);
+
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.notify(FOREGROUND_SERVICE_ID, notification);
+    }
+
+
```
Comment (VadymLuchko) [inline app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: redundant empty line

Comment (AKalinich-Luxoft) [reply app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: @VadymLuchko fixed in https://github.com/CustomSDL/sdl_android_launcher/pull/5/commits/a71426ea4cf4330474712b02a694fff1a2356020

File: app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java (hunk 1 above)
Comment (AKalinich-Luxoft) [reply app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: @VadymLuchko thank you :)
I used an approach from this comment - https://stackoverflow.com/a/50888586
Take a look at https://github.com/CustomSDL/sdl_android_launcher/pull/5/commits/b172d24a3aeffb2fe7cb183d547e11b090ea66f4

Comment (VadymLuchko) [reply app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: hope this works

**Review by VadymLuchko — APPROVED** [review]

File: app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java (hunk 1 above)
Comment (AKalinich-Luxoft) [reply app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java]: @VadymLuchko it is. I checked :)

**Review by ychernysheva — APPROVED** [review]

**Review by YaroslavLutsenko — APPROVED** [review]

**Review by YaroslavLutsenko — APPROVED** [review]

**Pull request merged**